hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/PrisonerSynchroniserService.kt

```diff
@@ -93,7 +120,25 @@ class PrisonerSynchroniserService(
       isUpdated
     } ?: run {
       // Need to create, which means calling all domain data too
-      reindex(ob, listOf(SyncIndex.RED), eventType)
+
+      val prisoner = Prisoner().translate(
+        existingPrisoner = null,
+        ob = ob,
+        incentiveLevel = runCatching { getIncentive(ob) },
+        restrictedPatientData = runCatching { getRestrictedPatient(ob) },
+      )
+      prisonerRepository.save(prisoner, SyncIndex.RED)
+      prisonerMovementsEventService.generateAnyEvents(null, prisoner, ob, red = true)
+      alertsUpdatedEventService.generateAnyEvents(null, prisoner, red = true)
+
+      // simulating domainEventEmitter.emitPrisonerCreatedEvent(ob.offenderNo)
+      telemetryClient.trackPrisonerEvent(
+        TelemetryEvents.RED_SIMULATE_PRISONER_CREATED_EVENT,
+        prisonerNumber = ob.offenderNo,
+        bookingId = ob.bookingId,
+        eventType = eventType,
+      )
+
```

Comment:
Cannot use reindex() as that has real event generation included. This code does the same job but with red-specific events and logging